scripts/find_sick

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env python
+
+# Software License Agreement (BSD) 
+#
+# @@Mustafa Safri    msafri@clearpathrobotics.com
+# @@copyright (c) @(2015), Clearpath Robotics, Inc., All rights reserved.
```

Comment:
Ah, what you copied in was the template. Please fix these headers, eg: https://github.com/clearpathrobotics/robot_upstart/blob/jade-devel/scripts/install